Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12 as builder
+WORKDIR /go/src/github.com/cloudflare/cloudflared/
+ADD . .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o cloudflared ./cmd/cloudflared
```

Comment:
if you build with cgo disabled, compression will no longer work.  Compression relies on a cgo brotli implementation.

Reply:
thanks Matt. I'll update it. I guess  that also means we can't use alpine :smile:  since last I checked musl didn't work well with cgo